frontend/lib/main.dart

```diff
@@ -12,6 +12,7 @@ class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
+      debugShowCheckedModeBanner: false,
```

Comment:
Uhm, eh? @AliceCasali

Reply:
This is needed in order to remove the debug icon in the top right corner of the screen. I cannot see my close icon without this command